### DISCUSSION TITLE
Windows Server compatibility

### DISCUSSION THREAD

**Discussion by mzzy-x** [Klocman/Bulk-Crap-Uninstaller#312] [Ideas]
Hey, the installer does not go through on Windows Server 2019. Could you make BCU compatible ? Thanks. All the best

**Comment by Klocman** [comment]
What exactly seems to be the issue? Does the portable version work? BCU did
work fine on previous versions of win server.

**Comment by mzzy-x** [comment]
Hey, thanks for getting back
So the installation completes fine but when the exe is launched, but also the portable version, I was getting an error. Sadly, I uninstalled Windows Server 2019

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by mzzy-x** [comment]
I can't tell you which error it was since I uninstalled Windows Server 2019 for Windows 10 since then. Sorry!

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
